README.md

```diff
@@ -26,19 +26,19 @@ If you get stuck or you are starting out and want more details, the rest of this
 * [Creating admin users for your cluster](#optional-create-dedicated-and-cluster-admins)
 * [Cleaning up](#next-steps)
 
-By the end of this guide you will have an Amazon Red Hat OpenShift cluster running in your AWS account.
+By the end of this guide you will have an Red Hat OpenShift Service on AWS cluster running in your AWS account.
```

Comment:
By the end of this guide you will have **a** Red Hat OpenShift Service on AWS cluster running in your AWS account.

Reply:
Good catch, thanks! Should be fixed now